src/modules/Forum/components/ForumItemPreview/ForumItemPreview.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+import { block } from 'bem-cn';
+
+const b = block('forum-item-preview');
+
+type ForumItemPreviewProps = {
+  className?: string;
+  descSlot?: ReactNode;
+  statSlot?: ReactNode;
+};
+
+function ForumItemPreview({
+  descSlot,
+  statSlot,
+  className = '',
+}: ForumItemPreviewProps): JSX.Element {
```

Comment:
🧐
Так ведь легче читается и объявление функции занимает ровно одну строчку)
```suggestion
function ForumItemPreview(props: ForumItemPreviewProps): JSX.Element {
const {
  descSlot,
  statSlot,
  className = '',
} = props;
```

Reply:
Мне норм , это случайно не ещё 1 вопрос для стайлгайда? Кстати не считаю, что объявление функции одной строкой против нескольких в большинстве подобных случаев несёт доп. пользу. Тут полезная инфа — только имя функции, и сами пропы, которые как раз ближе к тебе в мультилайн-варианте. Возврат функции всегда один и тот же, и он даже вроде опциональный для написания

Reply:
("подобных случаев" — функциональный компонент с пропсами)

Reply:
Всё так, но так у тебя все функции будут выглядеть одинаково, а единообразие — это хорошо)
Тут доп. польза именно в этом + ты явно видишь тип пропсов, а не ищешь их несколькими строчками ниже. В общем, это всё безусловно вкусовщина, поэтому да, можно вынести на обсуждение как вопрос стайлгайда

Reply:
Единообразие — значит никогда не использовать деструктуризацию в аргументах функции, это уже довольно громкое заявление, я бы сказал самое жёсткое правило из всего стайлгайда. И не без своих минусов, `const` и доп. переменная `props` добавляют визуальный шум. Тип пропсов всегда определяется перед самой функцией, т.е. виден, но это тоже вопрос конвенции @mukhindev @DariaZherebtsova

Reply:
Под шумом я понимал лишние символы/конструкции

Reply:
@andreyvolokitin @kotosha-real
Мне отдельно диструтизировать пропс нравится больше, меньше вложенных скобок.
Хотя если честно сам не придерживаюсь единообразия в этом вопросе. Я больше руководствуюсь количеством пропсов. Если их 2-3, делаю одной строкой прямо в аргументах, если много, выношу диструкторизацию из скобок.